### DISCUSSION TITLE
ios客户端假死

### DISCUSSION THREAD

**Discussion by rhymesgeckos0h** [cloudreve/ios-feedback#27] [General]
操作系统：ipados 17.4.1
软件版本：cloudreve 3.0.2 （app store显示版本）
前置操作：将音频暂停后挂到后台
注意：无法稳定复现
视频：https://player.dogecloud.com/web/player.html?vcode=51ff3aa125cf3761&userId=2159&autoPlay=false
论坛链接：https://forum.cloudreve.org/d/4331

**ios客户端发现了很多假死、操作不顺手、离谱的操作（我是ipad，我不喜欢苹果手机）。我不知道在iphone上是什么样的情况，但是就ipad版本我是很不愿意在pro版本的基础上再付费400的。作为一个商业软件，cloudreve ios的表现实在不行。**

**Comment by vvisionnn** [comment]
since you say you met many error but I can only get one (which is this issue), how about report them all so I can fix them all

I will convert this as discussion since there is no repro step provide

**Reply by rhymesgeckos0h** [reply]
您说的对，我为我之前的无礼感到抱歉。之后我将会在这里反馈我发现的问题。由于我只是一个用户，所以反馈会非常主观。请您理解。
由于我学习的外语语种并非英语。我将使用翻译软件将中文翻译成英文。但请注意：翻译后的内容仅供参考。

You are right, and I apologize for my previous rudeness. Moving forward, I will provide feedback on the issues I encounter here. As I am merely a user, my feedback will inevitably be quite subjective. Please understand that. Since English is not the foreign language I am studying, I will be using translation software to convert my Chinese text into English. However, please note that the translated content is for reference only.

**Comment by rhymesgeckos0h** [comment]
> since you say you met many error but I can only get one (which is this issue), how about report them all so I can fix them all
> 
> I will convert this as discussion since there is no repro step provide

软件选择在标签页面时，将软件放置于后台后再次切换会导致无法呼唤出菜单。这是我觉得最难受的一点。
操作系统：ipados 17.4.1
软件版本：cloudreve 3.0.2 （app store显示版本）
注意：就目前而言，可以稳定复现。
视频：
- https://player.dogecloud.com/web/player.html?vcode=1751d1c8f53ed5f0&userId=2159&autoPlay=false
- https://player.dogecloud.com/web/player.html?vcode=a0ed210ff960c44d&userId=2159&autoPlay=false

When selecting an app from the multitasking menu, putting the app in the background and then switching back to it subsequently causes the inability to summon the menu. This is the most frustrating aspect for me.
Operating System: iPadOS 17.4.1
Software Version: Cloudreve 3.0.2 (as displayed on the App Store)
Note: Currently, this issue can be reliably reproduced.
Video: 
- https://player.dogecloud.com/web/player.html?vcode=1751d1c8f53ed5f0&userId=2159&autoPlay=false
- https://player.dogecloud.com/web/player.html?vcode=a0ed210ff960c44d&userId=2159&autoPlay=false

**Comment by rhymesgeckos0h** [comment]
用户名过长和设置图片重叠：
![wOX51Spm36970568-d49b-4355-b514-c3fede0149e7iVUGcDoVGgNrxNey_IMG_0791](https://github.com/cloudreve/ios-feedback/assets/168970824/f6d4750e-3bb6-41ce-8932-c58659d49a88)


Username too long and profile picture overlapping.
![wOX51Spm36970568-d49b-4355-b514-c3fede0149e7iVUGcDoVGgNrxNey_IMG_0791](https://github.com/cloudreve/ios-feedback/assets/168970824/f6d4750e-3bb6-41ce-8932-c58659d49a88)

**Comment by rhymesgeckos0h** [comment]
我是通过站点付费统一购买的，发现在后台设置“用户论坛url”、"反馈页面url"中，只要有一项是空的则在ios客户端中就都不显示。我希望能够是，如果哪一项空着，则不显示哪一项。

I purchased through unified payment on the site and found that in the backend settings for "User Forum URL" and "Feedback Page URL", if any one of these fields is empty, neither of them shows up in the iOS client. I would like the system to behave such that only the field which is empty is not displayed, while the other, if filled, remains visible.

**Comment by rhymesgeckos0h** [comment]
貌似通过ios客户端可以创建分享链接，但是不能查看已经分享的内容和取消分享。导致手滑如何触发分享还需要到网页版去取消。我希望在ios上可以查看分享和取消分享。

It seems that while it's possible to create share links via the iOS client, there's no functionality to view already shared content or to undo shares. This means that if a share is accidentally triggered, one has to go to the web version to cancel it. I am hoping for the ability to view and cancel shares directly on iOS.

**Comment by rhymesgeckos0h** [comment]
我发现在通过ios上传大文件的时候不是很友好：虽然在右上角有传输进度，但是没有明显提示。并且如果上传失败，也不会有明显的提示。我认为，是不是可以做一个动画在上传文件的时候，引导用户观察到右上角传输进度的二级菜单。并且传输进度的图标也可以像web端一样，依照上传的总体进度对圆形图标边缘加粗表示进度。对于上传失败的情况，将上传失败的原因用类似软件里切换账户的弹窗一样在软件上方提示一下用户。

I have found that uploading large files via iOS isn't very user-friendly: although there's a transfer progress indicator in the top right corner, it lacks clear notification. Moreover, if the upload fails, there isn't an obvious alert either. I suggest implementing an animation during file uploads to guide users to notice the secondary menu with the progress detail in the top right corner. The progress icon could also mimic the web version by thickening the edge of the circular icon according to the overall upload progress, thereby visually representing the progress. In cases of upload failure, the reason for the failure should be presented to users in a pop-up, similar to how switching accounts is handled in the app, appearing at the top of the software.

**Comment by rhymesgeckos0h** [comment]
ios客户端好像不可以查看gif的动图（gif图片点击进去就只有静态内容。）视频：https://player.dogecloud.com/web/player.html?vcode=0202d148f5d7d9a4&userId=2159&autoPlay=false

It seems that the iOS client does not support viewing animated GIFs (when a GIF image is clicked, it only displays static content). As for videos: https://player.dogecloud.com/web/player.html?vcode=0202d148f5d7d9a4&userId=2159&autoPlay=false
